scan/proxy/default.vcl

```diff
@@ -5,14 +5,30 @@ backend default {
   .port = "1317";
 }
 
+backend bandsv {
+  .host = "172.18.0.16";
+  .port = "5000";
+}
+
 sub vcl_deliver {
   if (req.url ~ "/") {
     set resp.http.Access-Control-Allow-Origin = "*";
-    set resp.http.Access-Control-Allow-Methods = "GET, OPTIONS";
+    set resp.http.Access-Control-Allow-Methods = "GET, OPTIONS, POST, PATCH, PUT, DELETE";
     set resp.http.Access-Control-Allow-Headers = "Origin, Accept, Content-Type, X-Requested-With, X-CSRF-Token";
   }
 }
 
+sub vcl_recv {
+  if (req.url ~ "^/bandsv/") {
+    set req.url = regsub(req.url, "^/bandsv", "/");
+    set req.backend_hint = bandsv;
+  } else {
+    set req.backend_hint = default;
+  }      
+}
+
 sub vcl_backend_response {
+  if (!(bereq.url == "^/bandsv/")) {
```

Comment:
How did we confirm this does the job we want? It's weird that the logic here is different from line 22.

Reply:
As I tested, remove this condition that works correctly